auth/src/main/java/org/aerogear/android/ags/auth/AuthService.java

```diff
@@ -75,7 +79,7 @@ public void setAuthenticatorChain(AuthenticationChain newChain) {
     public static synchronized AuthService getInstance() {
         if (INSTANCE == null) {
             // FIXME: load the configurations from core and pass it here
-            INSTANCE = new AuthService(null);
+            INSTANCE = new AuthService();
```

Comment:
We should NOT be using singletons.  References to services should be fetch through the mobile core and not plucked from the ether.

See https://github.com/aerogear/proposals/blob/master/android-sdk/README.md#referencing-a-service

Reply:
Yeah I don't think this is used anyway, doesn't seem to be referenced anywhere. Must have been left over. Will remove